src/fairseq2/recipes/wav2vec2/asr/train.py

```diff
@@ -188,6 +188,12 @@ class Wav2Vec2AsrTrainConfig:
     """The number of checkpoints to keep based on their validation score. If
     ``None``, none will be deleted."""
 
+    score_metric_name : str = "uer"
+    """The validation metric to keep track of."""
+
+    lower_better: bool = True
```

Comment:
Seems like lower better is always going to be True, we can skip this? Unless you can foresee scenarios where this will change.